src/plugins/add-cal/reports/a11y-1-fr.hbs

```diff
@@ -0,0 +1,75 @@
+---
+{
+	"title": "Add to Calendar",
+	"language": "fr",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "add-cal",
+	"parentdir": "add-cal",
```

Comment:
Add 2 front-matter in order to fix the broken link found by our continuous integration tools.

```suggestion
	"parentdir": "add-cal",
	"parentPage": "Ajout au calendrierr",
	"parentPageURL": "add-cal",
```

Reply:
Commenting to resolve conversation, changes were applied in previous commit